docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+
+Regarding the components, usually they are automatically selected by the
+different tools that are behind JIRA, but in case that we found a JIRA
+ticket without the component the available values are the following:
+
+
+
+| **Components** | **Description** | 
+| -- | -- | 
+| FIWARE-COLLABORATION-REQ | Issues related to the request associated to some type of collaboration with FIWARE in terms of participation in some events or in terms of improvement some FIWARE GEri |
+| FIWARE-FEEDBACK | General issues related to feedback recover from the users. |
+| FIWARE-GENERAL-HELP | Issues in general not classified in the other components. |
+| FIWARE-LAB-HELP | Issues related to some of the FIWARE Lab nodes |
+| FIWARE-MUNDUS-REQ | Issues regarding the activities of FIWARE associated to the extension of FIWARE beyond Europe (FI-GLOBAL project). |
+| FIWARE-OPEN-DATA-REQ | (Deprecated) Issues related to the management of Open Data inside CKAN tool or related to the possibility to contribute with Open Data inside FIWARE ecosystem. |
+| FIWARE-OPS-HELP | Issues related to the use of the different FIWARE Ops tools.
+| FIWARE-SMART-CITIES-REQ | (Deprecated) Issues related to the collaboration in the SmartCities |
+| FIWARE-SPEAKERS-REQ | Issues related to the request to get some FIWARE expert to provide some presentation or speech in some events or just in some Summer School or so on. |
+| FIWARE-TECH-HELP | Issues related to some of the FIWARE GEri. |
+| FIWARE-TRAINING-REQ | (Deprecated) Issues related to the request of training for some of the accelerator program. Currently, it is not needed due to there is no request for training from accelerator programs. |
+
+In case of HD-Chapter and HD-Enabler, they make reference to the
+corresponding Chapter and Enabler that have to be assigned the
+corresponding JIRA ticket, how it was described previously (see Figure
+6.4: HD-Chapter, HD-Enabler and HD-Node attributes in issues.). It is
+something that have to be selected once we edit the corresponding JIRA
+issue.
+
+![Jira Help Desk issue.](image23.png)
+
+Last but not least, HD-Node makes reference to the corresponding FIWARE
+Lab node in which this issue should be resolved. Usually, the issues are
+related to FIWARE GEis or FIWARE Lab nodes. The first one is associated
+to the email <fiware-tech-help@lists.fiware.org> and the second is
+associated to <fiware-lab-help@lists.fiware.org>. This means:
+
+-   FIWARE GEis have associated the attribute of the HD-Enabler in order
+    to identity which one is it and the HD-Chapter to know on which
+    FIWARE Chapter is included this enabler.
+
+-   FIWARE Lab nodes has associated the attribute HD-Node, in order to
+    know to which FIWARE Lab node is assigned this issue or Jira ticket.
+
+-   Due to these tickets come from different source it is needed to
+    complete only HD-Chapter and HD-Enabler in case of tickets
```

Comment:
complete only HD-Enabler in case of tickets associated to FIWARE GEs or HD-Node is case of tickets associated to FIWARE Lab nodes.